### PR TITLE
fix(debugging): use millisecond for duration in expression DSL

### PR DESCRIPTION
**Pull request by P403n1x87**
We convert the `@duration` injected local variable in milliseconds to match the UI expectation and report correct numbers.

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed. If no release note is required, add label `changelog/no-changelog`.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.
- [x] Reviewer has explicitly acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment.
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)

### CODE REVIEW

File: releasenotes/notes/fix-debugger-duration-ms-in-dsl-aad91c0b730bbdf0.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    dynamic instrumentation: function durations are now reported in milliseconds
+    to match the expectation from the UI.
```
Comment (P403n1x87) [inline releasenotes/notes/fix-debugger-duration-ms-in-dsl-aad91c0b730bbdf0.yaml:5]:
```suggestion
    dynamic instrumentation: function duration measurements are now reported in
    milliseconds to match the expectation from the UI.
```

**Review by shatzi — APPROVED** [review]
should we add a unit-test?

**Comment by P403n1x87** [comment]
> should we add a unit-test?

I was thinking this is something good for system tests, but there might be a way to test this easily to quickly catch regressions. I'll have a look

**Comment by P403n1x87** [comment]
@shatzi I've added a test case

**Comment by ZStriker19** [comment]
Duration was in nanoseconds before? Seems good!

**Review by ZStriker19 — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**